ArmPkg/Library/DebugPeCoffExtraActionLib/DebugPeCoffExtraActionLib.c

```diff
@@ -32,7 +32,7 @@ PeCoffLoaderRelocateImageExtraAction (
   IN OUT PE_COFF_LOADER_IMAGE_CONTEXT  *ImageContext
   )
 {
-#ifdef __GNUC__
+ #ifdef __GNUC__

```

Comment:
nit: i dont think you need to change this. the tool is too nice to run on all files :)